autogen.sh

```diff
@@ -4,7 +4,6 @@ set -x
 
 autopoint --force || exit 1
 libtoolize --automake --copy --force || exit 1
-intltoolize --copy --force || exit 1
 aclocal -I m4 --force || exit 1
 autoheader --force || exit 1
 automake --add-missing --copy --force || exit 1
```

Comment:
Probably I'd change the order of automake and autoconf.
I think automake is better to be run after autoconf.

Reply:
Why not switching to `autoreconf` then? What would you think of using the [GNOME autogen.sh template](https://wiki.gnome.org/Projects/GnomeCommon/Migration)?

Reply:
You're right. autoreconf looks better and using the GNOME autogen.sh is also fine with me.